spotbugsTestCases/src/java/infiniteLoop/Issue2331.java

```diff
@@ -0,0 +1,11 @@
+package infiniteLoop;
+
+public class Issue2331 {
+
+    public void loop(){
```

Comment:
There should be a space before the brace.

Reply:
@ThrawnCA  could you please merge this request as I am do not have write access?